### DISCUSSION TITLE
Error when setting up grammarly

### DISCUSSION THREAD

**Discussion by asher-gh** [williamboman/nvim-lsp-installer#805] [Q&A] [answered]
Using the nvchad config but getting this error when adding the grammarly server
```
Error executing vim.schedule lua callback: ...w/Cellar/neovim/0.7.0/share/nvim/runtime/lua/vim/
lsp.lua:955: RPC[Error] code_name = InternalError, message = "Request initialize failed with me
ssage: clientId is required"
stack traceback:
        [C]: in function 'assert'
        ...w/Cellar/neovim/0.7.0/share/nvim/runtime/lua/vim/lsp.lua:955: in function ''
        vim/_editor.lua: in function <vim/_editor.lua:0>
```

**Comment by williamboman** [comment, marked as answer]
Hello! This doesn't really belong in this repository, I'd suggest seeking help upstream (with the Grammarly LSP maintainers directly). Even better would be to try to get to the bottom of what might be wrong yourself, and come up with a fix and contribute it to [lspconfig](https://github.com/neovim/nvim-lspconfig)!
